<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginWindow</class>
 <widget class="QMainWindow" name="LoginWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>450</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>450</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>450</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>[LoginWindow.windowTitle]</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resources.qrc">
    <normaloff>:/icons/lge_perplexity2_24x24.png</normaloff>:/icons/lge_perplexity2_24x24.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">/* --- Загальний фон --- */
QMainWindow {
    background-color:#ddeaff;
}

/* --- Ліва панель із логотипом --- */
QWidget#leftPanel {
    background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1,
        stop:0 #1A2A3A, stop:1 #101820);
}

/* --- Напис під логотипом --- */
QLabel#lblTitle {
    color: #CCCCCC;
    font-size: 12pt;
    font-weight: bold;
}

/* --- Заголовок реєстрації --- */
QLabel#lblHeader {
    color: #333333;
    font-size: 14pt;
    font-weight: bold;
}

/* --- Поля введення --- */
QLineEdit {
    border: 1px solid #AAAAAA;
    border-radius: 4px;
    padding: 4px;
    font-size: 10pt;
}

/* --- Кнопки --- */
QPushButton {
    border-radius: 6px;
    height: 30px;
}

/* --- Кнопка Зареєструватися --- */
QPushButton#btnLogin {
    background-color: #2E7EF9;
    color: white;
}
QPushButton#btnLogin:hover {
    background-color: #1E5FD0;
}

/* --- Кнопка Вихід --- */
QPushButton#btnExit {
    background-color: #E0E0E0;
    color: #000000;
}

/* --- Текст помилок --- */
QLabel#lblError {
    color: red;
    font-size: 9pt;
    font-style: italic;
}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>800</width>
     <height>450</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>800</width>
     <height>450</height>
    </size>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="3,4">
    <item>
     <widget class="QWidget" name="leftPanel" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>330</width>
        <height>430</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>800</width>
        <height>500</height>
       </size>
      </property>
      <property name="baseSize">
       <size>
        <width>300</width>
        <height>400</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background: qlineargradient(x1:0, y1:0, x2:1, y2:0, stop:0 #2C7A8C, stop:1 #195D6F);</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="lblLogo">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>150</width>
             <height>200</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>330</width>
             <height>400</height>
            </size>
           </property>
           <property name="baseSize">
            <size>
             <width>300</width>
             <height>400</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap resource="../resources.qrc">:/icons/robot_600x600.png</pixmap>
           </property>
           <property name="scaledContents">
            <bool>true</bool>
           </property>
           <property name="alignment">
            <set>Qt::AlignmentFlag::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lblTitle">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>40</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Segoe UI</family>
             <pointsize>12</pointsize>
             <italic>false</italic>
             <bold>false</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">font: 12pt &quot;Segoe UI&quot;;</string>
           </property>
           <property name="text">
            <string>[LoginWindow.lblTitle]</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Orientation::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="rightPanel" native="true">
      <property name="styleSheet">
       <string notr="true">background: qlineargradient(x1:0, y1:0, x2:1, y2:0, stop:0 #2C7A8C, stop:1 #195D6F);</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,0,1,1">
         <property name="spacing">
          <number>4</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>6</number>
         </property>
         <property name="bottomMargin">
          <number>6</number>
         </property>
         <item>
          <widget class="QLabel" name="lblHeader">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>600</width>
             <height>30</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">font: 700 14pt &quot;Segoe UI&quot;;</string>
           </property>
           <property name="text">
            <string>[LoginWindow.lblHeader]</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignmentFlag::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QFormLayout" name="formLayout">
           <property name="horizontalSpacing">
            <number>20</number>
           </property>
           <property name="verticalSpacing">
            <number>60</number>
           </property>
           <property name="leftMargin">
            <number>20</number>
           </property>
           <property name="topMargin">
            <number>130</number>
           </property>
           <property name="rightMargin">
            <number>6</number>
           </property>
           <property name="bottomMargin">
            <number>130</number>
           </property>
           <item row="0" column="0">
            <widget class="QLineEdit" name="lineEdit">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>260</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>360</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="echoMode">
              <enum>QLineEdit::EchoMode::Password</enum>
             </property>
             <property name="placeholderText">
              <string>[LoginWindow.lineEdit]</string>
             </property>
             <property name="clearButtonEnabled">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLabel" name="lblPassword">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>28</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text">
              <string>[LoginWindow.lblPassword]</string>
             </property>
             <property name="scaledContents">
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QLabel" name="lblError">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>600</width>
             <height>20</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Segoe UI</family>
             <pointsize>10</pointsize>
             <italic>false</italic>
             <bold>false</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">font: 10pt &quot;Segoe UI&quot;;</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item alignment="Qt::AlignmentFlag::AlignHCenter">
            <widget class="QPushButton" name="btnLogin">
             <property name="text">
              <string>[LoginWindow.btnLogin]</string>
             </property>
            </widget>
           </item>
           <item alignment="Qt::AlignmentFlag::AlignHCenter">
            <widget class="QPushButton" name="btnExit">
             <property name="text">
              <string>[LoginWindow.btnExit]</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../resources.qrc"/>
  <include location="../resources.qrc"/>
  <include location="../resources.qrc"/>
 </resources>
 <connections/>
</ui>
